app/assets/javascripts/components/story/StoryNotes.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Note from 'components/notes/Note';
+import NoteForm from 'components/notes/NoteForm'
+
+class StoryNotes extends React.Component {
+  constructor() {
+    super();
+
+    this.renderNotes = this.renderNotes.bind(this);
+    this.renderStoryNotes = this.renderStoryNotes.bind(this);
+  }
+
+  render() {
+    return this.props.isNew ? null : this.renderStoryNotes();
+  }
+
+  renderStoryNotes() {
+    const { notes, disabled } = this.props;
+    return (
+      <div className="form-group">
+        <label htmlFor="notes"> { I18n.t('story.notes') } </label>
+        <div className="notelist">
+          { this.renderNotes() }
+        </div>
+        { !disabled && <NoteForm note={notes.last()} /> }
+      </div>
+    );
+  }
+
+  renderNotes() {
+    const { notes, disabled, handleDelete } = this.props;
+    const noteList = (disabled) ? notes : notes.slice(0, -1);
+    return noteList.map((note, i) =>
+      <Note
+        note={note}
+        disabled={disabled}
+        handleDelete={handleDelete}
+        key={i}
+      />
+    );
+  }
+
+  componentWillMount() {
```

Comment:
Why do we need it? Props should be treated as immutable so it's not a food idea to change it inside the component. If you need to have an empty note to be available do it _before_ passing the prop to this component.